session/role_integ_test.go

```diff
@@ -56,38 +56,38 @@ func Test_Integ_Role(t *testing.T) {
 	bob, err := session.New(bobCfg)
 	require.NoErrorf(t, err, "initializing bob session")
 	t.Logf("bob session id: %s\n", bob.ID())
-	t.Logf("alice database dir is: %s\n", aliceCfg.DatabaseDir)
+	t.Logf("bob database dir is: %s\n", bobCfg.DatabaseDir)
 
 	var alicePeerID, bobPeerID perun.PeerID
 	t.Run("GetPeerID", func(t *testing.T) {
 		t.Run("happy", func(t *testing.T) {
 			alicePeerID, err = alice.GetPeerID(perun.OwnAlias)
-			require.NoErrorf(t, err, "Alice: GetPeerID")
+			require.NoErrorf(t, err, "alice reading own peer ID")
 			alicePeerID.Alias = aliceAlias
 
 			bobPeerID, err = bob.GetPeerID(perun.OwnAlias)
-			require.NoErrorf(t, err, "Bob: GetPeerID")
+			require.NoErrorf(t, err, "bob reading own peer IDetPeerID")
 			bobPeerID.Alias = bobAlias
 		})
 		t.Run("missing", func(t *testing.T) {
 			_, err = alice.GetPeerID("random alias")
-			assert.Errorf(t, err, "Alice: GetPeerID")
+			assert.Errorf(t, err, "alice reading random peer ID should error")
 			t.Log(err)
 		})
 	})
 
 	t.Run("AddPeerID", func(t *testing.T) {
 		t.Run("happy", func(t *testing.T) {
 			err = alice.AddPeerID(bobPeerID)
-			require.NoErrorf(t, err, "Alice: AddPeerID")
+			require.NoErrorf(t, err, "alice adding bob's peer ID")
 
 			err = bob.AddPeerID(alicePeerID)
-			require.NoErrorf(t, err, "Bob: GetPeerID")
+			require.NoErrorf(t, err, "bob adding alice's peer ID")
 		})
 		t.Run("already_exists", func(t *testing.T) {
 			// Try to add bob peer ID again
 			err = alice.AddPeerID(bobPeerID)
-			assert.Errorf(t, err, "Alice: AddPeerID")
+			assert.Errorf(t, err, "alice adding bob's peer ID when it is already added should error")
```

Comment:
```suggestion
			assert.Errorf(t, err, "alice adding bob's peer ID should error when it is already added")
```

Reply:
I would prefer to keep `should error` phrase at end of the sentence to provide more emphasis. However, I am changing `is already added` to `is already present`.